analyses/cnv-chrom-plot/cn_status_heatmap.Rmd

```diff
@@ -441,7 +443,7 @@ Save to PDF.
 
 ```{r}
 # Save plot as PDF
-pdf(file.path(plots_dir, "cn_status_heatmap.pdf"))
+png(file.path(plots_dir, "cn_status_heatmap.png"))
```

Comment:
Also, have you tried using the `res` argument of the `png` function with this plot as yet? The heatmap, as a `PNG`, is not as clear as we would necessarily want it I _think_ (the labels are a little difficult to read). I do not believe that the quality now is terrible but if we can improve it a bit I think it's worth improving.

Reply:
I did play with the `res` argument a bit. I don't think I pushed that change though. Will do that.